src/Microsoft.AspNet.Hosting/WebHostConfiguration.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.AspNet.Hosting
+{
+    public class WebHostConfiguration
```

Comment:
Perhaps consider making this a POCO and just using ConfigurationBinder to directly bind against config?